examples/hacker_news_assets/tox.ini

```diff
@@ -8,13 +8,14 @@ setenv =
 passenv = CI_* COVERALLS_REPO_TOKEN BUILDKITE SNOWFLAKE_ACCOUNT SNOWFLAKE_USER SNOWFLAKE_PASSWORD
 deps =
   -e ../../python_modules/dagster[mypy,test]
+  -e ../../python_modules/dagit
+  -e ../../python_modules/dagster-graphql
```

Comment:
Why dagster-graphql?

Reply:
some of the dependencies here install `dagster-graphql` if we don't include it as an editable here, tox would install a released version which caused version mismatch in tests.

can audit if that's real necessary but don't think it's blocking the release

Reply:
gotcha sounds fine